docs/guides/wifi.md

```diff
@@ -1,117 +1,32 @@
-
-For **Ubuntu**, the "t2-j-bigsur", "t2-hwe-bigsur" and "t2-big-sur" variant kernels use Big Sur firmware too. You can get them from [here](https://github.com/marcosfad/mbp-ubuntu-kernel/releases) or this [alternative link](https://github.com/AdityaGarg8/T2-Big-Sur-Ubuntu-Kernel#pre-installation-steps). If you are a 16,1 or 16,2 user and want support for both bluetooth and wifi, then you will have to install a kernel available on the **alternative link** given above. Make sure that your [DKMS Modules](https://wiki.t2linux.org/guides/dkms) are updated and you install **linux-headers** before **linux-image**.
-
-There are also kernels available for [Arch based distros](https://github.com/jamlam/mbp-16.1-linux-wifi/releases) (like Arch Linux and Manjaro) but you can [compile it yourself](https://wiki.t2linux.org/guides/wifi/#compiling-a-kernel-for-using-big-sur-firmware) if you need/want to.
-
-This firmware is available in `/usr/share/firmware/wifi` in macOS Big Sur installations, or online at [https://github.com/AdityaGarg8/macOS-Big-Sur-WiFi-Firmware](https://github.com/AdityaGarg8/macOS-Big-Sur-WiFi-Firmware). If you get firmware from macOS, make sure that if the files are aliases, that you right click on them, and select "Show Original" to get the actual files.
-
-If your model identifier is 16,1 you can take a shortcut and get the firmware files from [https://github.com/AdityaGarg8/mbp-16.1-wifi-firmware](https://github.com/AdityaGarg8/mbp-16.1-wifi-firmware) by following the instructions given there. If you unable to get working firmware files from there then follow the instructions given for other models.
-
-##### Compiling a kernel for using Big Sur firmware
-
-Follow the [kernel compiling guide](https://wiki.t2linux.org/guides/kernel/#compile),
-but make sure to use [https://github.com/jamlam/mbp-16.1-linux-wifi](https://github.com/jamlam/mbp-16.1-linux-wifi)
-instead of [https://github.com/aunali1/linux-mbp-arch](https://github.com/aunali1/linux-mbp-arch)
-as the patchset repository.
-
-### Renaming Firmware
-
-- Rename the `.trx` "Firmware" file to `brcmfmac4364-pcie.bin`, but change `4364` to your wifi chipset number.
-- Do the same for the `.clmb` "Regulatory" file and rename it to `brcmfmac4364-pcie.clm_blob`. Again, change `4364` to your chipset number.
-- Lastly, rename the `.txt` "NVRAM" file to `brcmfmac4364-pcie.Apple Inc.-MacBookPro15,1.txt` but in addition to changing `4364` to your chipset number, change `MacBookPro15,1` to your Model Identifier.
+You may download the firmware from [here](https://github.com/AdityaGarg8/Apple-WiFi-Firmware/releases/download/Montery/firmware.tar.xz).
 
 ### Installing Firmware
 
-1. Now that you got those 3 files, move them to `/lib/firmware/brcm/`.
-2. Check that the files are in place with `ls -l /lib/firmware/brcm|grep -E "43(64|55|77)"`. The output should look something like this
+1. Extract the firmware files from the .tar.xz and move them to `/lib/firmware/brcm/`.
```

Comment:
Given this will need to be done as root and as some file managers make it hard to elevate to root privileges, could you provide an example of commands needed to extract this to the correct place?